core/src/main/java/com/cloud/agent/api/GetStoragePoolCapabilitiesAnswer.java

```diff
@@ -0,0 +1,19 @@
+package com.cloud.agent.api;
+
+public class GetStoragePoolCapabilitiesAnswer extends Answer {
+    public GetStoragePoolCapabilitiesAnswer(GetStoragePoolCapabilitiesCommand cmd) {
+        super(cmd);
+
+        poolInfo = new StoragePoolInfo();
+    }
+
+    public StoragePoolInfo getPoolInfo() {
+        return poolInfo;
+    }
+
+    public void setPoolInfo(StoragePoolInfo poolInfo) {
+        this.poolInfo = poolInfo;
+    }
+
+    private StoragePoolInfo poolInfo;
```

Comment:
I think capability details map should ok in the response.

Reply:
Done